netty/src/main/java/io/grpc/netty/JettyTlsUtil.java

```diff
@@ -67,4 +96,17 @@ static synchronized Throwable getJettyNpnUnavailabilityCause() {
     }
     return jettyNpnUnavailabilityCause;
   }
+
+  /**
+   * Indicates whether Netty Java 9 ALPN is available.
+   */
+  static boolean isNettyJava9AlpnAvailable() {
+    try {
+      Class.forName("io.netty.handler.ssl.Java9SslEngine");
```

Comment:
We don't tend to do this sort of thing. We instead just upgrade to the netty that supports the feature we need.